.env.development

```
@@ -1 +1,4 @@
 NEXT_PUBLIC_API_HOST = http://127.0.0.1:8080
+
+# Skip OAuth proxy for local development (proxy server may be unavailable)
+SKIP_OAUTH_PROXY = 1
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)